core/templates/services/math-interactions.service.spec.ts

```diff
@@ -186,7 +186,7 @@ describe('MathInteractionsService', () => {
     // Failure cases.
     expect(mathInteractionsService.validateEquation('')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
-      'Please enter a non-empty answer.');
+      'Your answer seems to be empty.');
```

Comment:
"seems to be empty" is probably not right. Maybe you can say "Please enter an answer before submitting"

Reply:
Done.